NetworkPkg/Library/DxeHttpLib/DxeHttpLib.c

```diff
@@ -2077,6 +2077,8 @@ HttpMappingToStatusCode (
       return HTTP_STATUS_416_REQUESTED_RANGE_NOT_SATISFIED;
     case 417:
       return HTTP_STATUS_417_EXPECTATION_FAILED;
+    case 429:

```

Comment:
Where to these constant values such as 429 come from?  Is there are include file for the related HTTP specs or should one be added?

Reply:
Is this the official registry of these value?

https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml

Reply:
That's correct

Reply:
Can we add a reference to this registry in the function header.

Reply:
Sure.  I'll add a link to the official http status codes there.